### PR TITLE
:sparkles: Add python script to create ibus tables (closes #8)

### LINKED ISSUE #8: ibus table support
it'd be nice to release a ibus table versión of the autohotkey script. I don't think it'd be complicated and would help linux users who sadly don't have access to ahk. Having officially here would make it easier to have it consistently updated

If nobody wants, I can work on this tomorrow

### PR DESCRIPTION
**Pull request by Brian3647**
Using an IBus table is currently the preferred IME for linux users. Having it here will make it easier to be updated. This adds a python script that automatically converts the AHK scripts to simple ibus tables, based on the previous most used one for sitelen pona. The only pip dep needed is uuid.

The python code maybe could be improved but it works very well on the current ahk scripts. Do note that it won't add fancy ahk features if those are used in the future.

closes #8

### CODE REVIEW

**Comment by Brian3647** [comment]
some improvements this offers over the previous ibus table for sitelen pona:
- CJK-style quotes
- more combining glyphs
- logograph fullwidth space (both zz and ss)
- variation selectors

**Comment by Brian3647** [comment]
Optional TODO:

[ ] Change the name of ibus tables to the original ahk's file name

**Comment by Brian3647** [comment]
Optional TODO:

[ ] Change the name of ibus tables to the original ahk's file name

**Comment by Brian3647** [comment]
**BEFORE MERGING**

I can't rn but the `UUID = ..` should be inside the `gen_table` function and not outside

**Comment by Brian3647** [comment]
I think this is ready for review and merge

File: ahk-script/ibus-table-generator.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
+import os
+import uuid
+import datetime
+
+now = datetime.datetime.now().strftime("%Y%m%d")
+
+# taken and modified from here: https://github.com/neroist/sitelen-pona-ucsur-guide/blob/main/tokipona.txt
+# created by jan Komi and edited by authors of sitelen pona ucsur guide and kule Piton
```
Comment (neroist) [inline ahk-script/ibus-table-generator.py:8]: afaik jan Lili did not edit the table, majority of changes are from jan Kekan San and `@Qwerty-Space`. mi pali lili taso li pana e nimi su tawa ona

**Comment by neroist** [comment]
The ibus tables & script should be in a separate directory. Maybe the ahk scripts and tables could be moved to the UCSUR guide.

**Comment by Brian3647** [comment]
yes, I think the same - if you want, copy the ahk files and this python script to your repo and this pr will probably be closed which is OK. I'm not a fan of having the IME files in this repo etiher since they work for a lot of fonts - not only nasin nanpa.

**Comment by Brian3647** [comment]
yes, I think the same - if you want, copy the ahk files and this python script to your repo and this pr will probably be closed which is OK. I'm not a fan of having the IME files in this repo etiher since they work for a lot of fonts - not only nasin nanpa.

**Comment by neroist** [comment]
https://github.com/neroist/sitelen-pona-ucsur-guide/commit/6b10408193a4dffb8b14660356235c60daf76a47 :p

**Pull request closed without merging**